pycaching/cache.py

```diff
@@ -839,6 +863,7 @@ def load_quick(self):
         Use information from geocaching map tooltips. Therefore loading is very quick, but
         the only loaded properties are: `name`, `type`, `state`, `size`, `difficulty`, `terrain`,
         `hidden`, `author`, `favorites` and `pm_only`.
+        It also loads `status`, however locked caches are considered archived.
```

Comment:
This needs to be updated after recent adjustment.

Reply:
Updated in https://github.com/tomasbedrich/pycaching/pull/193/commits/432268b4ad5fd9e14a6f2e1411134ed2fb9427cf